task/max2max/internal/config/config.go

```diff
@@ -0,0 +1,32 @@
+package config
+
+import (
+	"github.com/aliyun/aliyun-odps-go-sdk/odps"
+)
+
+type Config struct {
+	*odps.Config
+	LogLevel           string
+	LoadMethod         string
+	QueryFilePath      string
+	DestinationTableID string
+}
+
+func NewConfig() *Config {
+	cfg := &Config{
+		Config: odps.NewConfig(),
+		// max2max related config
+		LogLevel:           getEnv("LOG_LEVEL", "INFO"),
+		LoadMethod:         getEnv("LOAD_METHOD", "APPEND"),
+		QueryFilePath:      getEnv("QUERY_FILE_PATH", ""),
+		DestinationTableID: getEnv("DESTINATION_TABLE_ID", ""),
+	}
+	// ali-odps-go-sdk related config
+	cfg.Config.AccessId = getEnv("ACCESS_ID", "")
+	cfg.Config.AccessKey = getEnv("ACCESS_KEY", "")
+	cfg.Config.Endpoint = getEnv("ENDPOINT", "http://service.ap-southeast-5.maxcompute.aliyun.com/api")
+	cfg.Config.ProjectName = getEnv("PROJECT", "")
```

Comment:
We are going to wrap all these properties in a single json. so that it is easy to store them like bigquery.
https://github.com/goto/optimus/blob/957bf8ab853688bedff8e68edf331b9106c992b0/ext/store/maxcompute/client.go#L24